docs/visual-testing/integrations/webdriverio.md

```diff
@@ -0,0 +1,154 @@
+---
+sidebar_label: WebdriverIO
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# WebdriverIO Integration
+
+## Introduction
+
+This guide requires an existing WebdriverIO project.<br />
+You can alternatively take a look to our [example repository](#example).
+
+Sauce Labs Visual provides an integration with [WebdriverIO](https://webdriver.io/) through a service that you can add to any existing WebdriverIO project.
+
+Sauce Labs Visual adds new commands to the WebdriverIO's `browser` object:
+- `browser.check()`: Takes a screenshot and send it to Sauce Labs Visual for comparison.
+- `browser.checkResults()`: Waits for diff calculations to complete and returns a summary of results.
+
+Sample usage:
```

Comment:
@kb-kerem
Where do we explain what the object looks like?

Reply:
Pushed a change to explain it in Advanced usage section. Also provided a link to there.